core/Thekdar.js

```diff
@@ -111,15 +120,49 @@ class Thekdar extends EventEmitter {
     }
     worker.setAddress(address);
     worker.create();
-    worker.on(this.handleWorkerMessage(worker));
+    worker.onChildMessage(this.handleChildMessage(worker));
+    // worker.onWorkerMessage(this.handleWorkerMessage(worker));
```

Comment:
@ojhaujjwal that method is used here, this code seems not updated, you can check master, :)